packages/oidc-client/package.json

```
@@ -41,6 +41,7 @@
     "@types/react-dom": "^18.2.17",
     "@typescript-eslint/eslint-plugin": "^6.13.2",
     "@typescript-eslint/parser": "^6.13.2",
+    "@vitejs/plugin-react": "^4.2.1",
```

Sometimes tests in oidc-client fails with `React is not defiend` error, so installed this to suppress the error.